Cargo.toml

```
@@ -97,3 +103,6 @@ tower-abci = { version = "0.7" }
 tendermint = { version = "0.31", features = ["secp256k1"] }
 tendermint-rpc = { version = "0.31", features = ["secp256k1", "http-client", "websocket-client"] }
 tendermint-proto = { version = "0.31" }
+
+# Using the IPC SDK without the `fil-actor` feature so as not to depend on the actor `Runtime`.
+ipc-sdk = { git = "https://github.com/consensus-shipyard/ipc-actors.git", default-features = false, branch = "fm-162-fil-actor-feature" }
```

Should we release a tag or point to `main` before merging?
```suggestion
ipc-sdk = { git = "https://github.com/consensus-shipyard/ipc-actors.git", default-features = false, branch = "main" }
```